src/columnstore/columnstore_metadata.cpp

```diff
@@ -120,9 +120,16 @@ string ColumnstoreMetadata::GetTablePath(Oid oid) {
 
 void ColumnstoreMetadata::GetTableMetadata(Oid oid, string &table_name /*out*/, vector<string> &column_names /*out*/,
                                            vector<string> &column_types /*out*/) {
+    // Clear out params before emplace.
+    column_names.clear();
+    column_types.clear();
+
```

Comment:
I understand that's true for the most case, but I did met unexpected situations in the past, cannot be too careful.
The ideal signature, IMO, is to use return value instead of out params, but unfortunately C++ doesn't support it.

Reply:
Let's change it to `D_ASSERT(column_names.empty())`

Reply:
Sounds good! Updated.